templates/etc/apache2/sites-available/debops__tpl_macros.j2

```diff
@@ -66,3 +96,13 @@
        ansible_local.pki.gnutls_version|d())
    else "0.0.0" }}
 {% endmacro %}{# ]]] #}
+
+{% macro indent(content, width=4, indentfirst=False) %}{# [[[ #}
```

Comment:
just a thought:  Would Jinja be interested in providing such a filter directly?  Has there been a discussion, i.e. bug report?